Makefile

```diff
@@ -25,14 +25,27 @@ GIT_COMMIT := $(shell git rev-parse HEAD 2> /dev/null || true)
 
 all: binary
 
-binary:
+binary: ${DEST}skopeo
+
+# Build a docker image (skopeobuild) that has everything we need to build.
+# Then do the build and the output (skopeo) shold appear in ${DEST} dir (or .).
+${DEST}skopeo: cmd/skopeo
+	docker build ${DOCKER_BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
+	-docker rm -f skopeobuild
+	docker run -v ${PWD}:/src/github.com/projectatomic/skopeo \
+		--name skopeobuild --env DEST=/ skopeobuildimage make binary-local
+	docker cp skopeobuild:/skopeo ${DEST}skopeo
```

Comment:
I don't have a strong feeling about `DEST` but removing it would clean some stuff up, yes.